src/app/pages/users/user-list/user-list.page.ts

```diff
@@ -10,13 +10,13 @@ import { UsersService } from 'src/app/providers/users.service';
   styleUrls: ['./user-list.page.scss'],
 })
 export class UserListPage implements OnInit {
-  user: User[] = [];
+  users: User[] = [];
   userSelected: User;
 
   constructor(private usersService: UsersService) {}
 
   async ngOnInit() {
-    this.user = await this.usersService.allStudent();
+    this.users = await this.usersService.allStudent();
```

Comment:
Rendiamo dinamica questa assegnazione. Utilizza un param `:type` (students | teachers) e in base a questo decidi se farti tornare tutti gli students, oppure gli insegnati.

Reply:
Ok, quindi faccio un controllo a monte?